src/main/webapp/app/exam/manage/exams/exam-checklist-component/exam-edit-workingtime-dialog/exam-edit-working-time-dialog.component.html

```diff
@@ -0,0 +1,46 @@
+<form name="workingTimeForm" (ngSubmit)="confirmUpdateWorkingTime()" #editForm="ngForm">
+    <div class="modal-header">
+        <h4 class="modal-title">
+            <span jhiTranslate="artemisApp.examManagement.editWorkingTime.title">Edit Exam Duration</span>
+        </h4>
+        <button type="button" class="btn-close" data-dismiss="modal" aria-hidden="true" (click)="clear()"></button>
+    </div>
+    <div class="modal-body d-flex flex-column">
+        <jhi-working-time-control
+            name="workingTimeSeconds"
+            durationLabelText="artemisApp.examManagement.editWorkingTime.label"
+            [(ngModel)]="workingTimeSeconds"
+            [showRelative]="false"
+            [disabled]="isLoading"
+        />
+
+        <hr />
+
+        <div>
+            <p jhiTranslate="artemisApp.examManagement.editWorkingTime.question" [translateValues]="{ title: exam.title }">Do you really want to edit the duration of the exam?</p>
+            <jhi-confirm-entity-name
+                confirmationText="artemisApp.examManagement.editWorkingTime.typeNameToConfirm"
+                [entityName]="exam.title!"
+                [disabled]="isLoading"
+                [(ngModel)]="confirmEntityName"
+                name="confirmEntityName"
+            />
+        </div>
+    </div>
+    <div class="modal-footer">
+        <button type="button" class="btn btn-secondary" data-dismiss="modal" (click)="clear()" [disabled]="isLoading">
+            <fa-icon [icon]="faBan"></fa-icon>&nbsp;<span jhiTranslate="entity.action.cancel">Cancel</span>
+        </button>
+        <button
+            id="confirm"
+            type="submit"
+            [ngClass]="['btn', 'btn-warning']"
```

Comment:
Let's be consistent; you declared your classes like this so far in this file
```suggestion
            class="btn btn-warning"
```

Reply:
Thanks, sure